LibNoDaveConnectionLibrary/PLCs/S7_xxx/MC7/Helper.cs

```diff
@@ -945,7 +945,7 @@ public static string GetFCPointer(byte b1, byte b2, byte b3, byte b4)
             switch (b1)
             {                
                 case 0x80:
-                    anf = "P#PE ";
+                    anf = "P#P ";

```

Comment:
Is this correct? (I don't know what PE was... and what is P for...)

Reply:
I think so, at least that's how it looks in my software:
![image](https://github.com/dotnetprojects/DotNetSiemensPLCToolBoxLibrary/assets/43485413/b7f20698-e3f9-4112-8f5d-24e10d9f0891)